src/test/java/uk/gov/companieshouse/api/strikeoffobjections/file/FileTransferApiClientUnitTest.java

```diff
@@ -91,6 +93,40 @@ public void testUpload_GenericExceptionResponse() {
         assertThrows(RestClientException.class, () -> fileTransferApiClient.upload(REQUEST_ID, file));
     }
 
+    @Test
+    public void testDelete_success() {
```

Comment:
could these be camelCase to match CH java standards

Reply:
camel cased